_pages/contact.md

```diff
@@ -7,13 +7,15 @@ description: Have a question or a problem with login.gov?
 <div class="bg-navy">
   <div class="container cntnr-wide px2 py3">
     <h1 class="m0 white">
-      {{ page.title }}
+      {% t pages.contact_us %}
     </h1>
   </div>
 </div>
 
 <div class="bg-white">
   <div class="container cntnr-wide px2 pt4 pb5">
-    First, check out answers to <a href="{{ '/help/' | relative_url }}">common questions</a>. Still have a question or a problem with the platform? Contact us at <a href="mailto:login-contact@gsa.gov?subject=login.gov">login-contact@gsa.gov</a>.
+    <p markdown="1">
+      {{ site.translations[site.lang]["contact_page"]["content"] | replace: 'site.baseurl', site.baseurl }}
```

Comment:
We may consider using `markdownify` instead of `markdown="1"`:

```{{ site.translations[site.lang]["contact_page"]["content"] | replace: 'site.baseurl', site.baseurl | mardownify }}```